nextjs-13/prisma/migrations/migration_lock.toml

```diff
@@ -0,0 +1,3 @@
+# Please do not edit this file manually
+# It should be added in your version-control system (i.e. Git)
+provider = "postgresql"
```

Comment:
Related to [this discussion](https://github.com/nimblehq/js-framework-benchmark/pull/79/files#r1203377631), for instance, this file shows that is missing an EOF line is missing

Reply:
Ahh, this is the file `prisma` automatically creates when I use `prisma generate`. Ok let me add the EOF :D

Reply:
Oke done!